docs/topics/developer-lightspeed/proc_configuring-developer-profile-settings.adoc

```diff
@@ -0,0 +1,53 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2025-02-26
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-developer-lightspeed-profile-settings_{context}"]
+= Configuring the {mta-dl-plugin} profile settings
+
+To run an analysis using {mta-dl-plugin}, you must configure a profile that contains all the necessary configurations for an analysis, such as source and target technologies and API key to connect to your chosen large language model (LLM). 
+
+.Prerequisites
+
+* You installed the {ProductShortName} extension version 8.0.0 in Visual Studio (VS) Code. 
+//need to check how the user provides LLM credentials and write a new proc if needed
+* You provided LLM credentials to enable generative AI for the {ProductShortName} extension in `settings.json` file. 
+* You installed the {ProductShortName} distribution version 8.0.0 in your system. 
+* You installed the latest version of Language Support for Java(TM) by Red Hat extension in VS Code.
+//check if users still need Java 17+ in their system since Language Support extension is already a req.
+* You installed Jave 17+ and Maven 3.9.9+ in your system. 
+* You opened a Java project in your VS Code workspace.
+
+.Procedure
+
+. Open the `Konveyor View Analysis` page in either of the following ways: 
++
+.. Click the screen icon on the `Konveyor Issues` pane of the {ProductShortName} extension. 
+.. Type `Ctrl + Shift + P` on the search bar to open the Command Palette and enter `Konveyor:Open Konveyor Analysis View`.
++
+. Click the settings button on the `Konveyor View Analysis` page to configure a profile for your project.
+The `Get Ready to Analyze` pane lists the follwoing basic configurations required for an analysis: 
++
+
+.Verification
+
+After you complete the profile configuration, close the `Get Ready to Analyze` pane. You can verify that your configuration works by running an analysis. See run an analysis.
+
+.{mta-dl-plugin}  profile settings
+[cols="20%,80%a",options="header",]
+|====
+|Profile settings |Description
+|Select profile|Create a profile that you can reuse for multiple analyses. The profile name is part of the context provided to the LLM for analysis.  
+|Configure label selector|A label selector filters rules for analysis based on the source or target technology.
+
+Specify one or more target or source technologies (for example, cloud-readiness). {mta-dl-plugin} uses this configuration to determine the rules that are applied to a project during analysis. 
+
+If you mentioned a new target or a source technology in your custom rule, you can type that name to create and add the new item to the list.
+
+[NOTE]
+====
+You must configure either target or source tehcnologies before running an analysis.
+====
+|Set rules|Enable default rules and Select your custom rule that you want {mta-dl-plugin} to use for an analysis. You can use the custom rules in addition to the default rules.
```

Comment:
I am not sure, but worth considering we may want to refer to the Analysis capabilties as "MTA" so that the doc portion for that is covering both "MTA" and "MTA with Developer Lightspeed".   Then use the "MTA with Developer Lightspeed" just for the Generative AI portions.